examples/gocean/eg6/README.md

```diff
@@ -0,0 +1,36 @@
+# PSyclone GOcean Example 6
+
+**Authors** J. Henrichs, Bureau of Meteorology
+
+## Introduction
+
+This is a very simple test that shows how to use the kernel data extraction
+support in PSyclone. It is a stand alone program that can be compiled
+and run.
+
+## Compilation
+You have to compile dl_esm_inf (which is included in external/dl_esm_inf)
+and one of the extraction libraries in lib/extract. 
+The documentation assumes that lib/extract/netcdf is used.
+Instructions for those are are given in the corresponding subdirectories.
```

Comment:
There's no README in lib/extract/netcdf - please could you add one.

Reply:
Done.